requirements.txt

```diff
@@ -1,5 +1,11 @@
-django==1.8
-djangorestframework==2.3.11
-django-nose==1.4
+Django==1.8
 django-hosts==1.2
-unittest2==1.1.0
+django-nose==1.4
+djangorestframework==2.3.11
```

Comment:
Yeah there were some missing. I believe they were PyYAML, dateutil, and nose. The only way I knew they were missing was because PyCharm gave imports a red underline if the package wasn't installed. After I installed the missing packages I just pasted my `pip freeze` here.